templates/commands/render_test.go

```diff
@@ -352,13 +491,21 @@ steps:
 				tempDirNamer: tempDirNamer,
 			}
 			r := &Render{
+				globalFlags: flags.GlobalFlags{
+					FlagNoPrompt: tc.flagNoPrompt,
+				},
+				hasTTY:             tc.hasTTY,
 				flagDest:           dest,
 				flagForceOverwrite: tc.flagForceOverwrite,
 				flagInputs:         tc.flagInputs,
 				flagKeepTempDirs:   tc.flagKeepTempDirs,
 				flagSpec:           "spec.yaml",
 				source:             "github.com/myorg/myrepo",
 			}
+
+			stdin, _, _ := r.Pipe()
+			stdin.WriteString(tc.stdinInput)
```

Comment:
Couldn't figure out a way to send multiple sets of inputs. I believe since scanner.Scan() in the prompt method reads the entire buffer, this no be possible.

Open to suggestions.

Reply:
I think you can do it in a goroutine, but `scanner.Scan` iterates over newlines, so you might be able to cheat and write something like "abc\ndef\njkl"

Reply:
I tried that, the first value is ok but the second one disappears.